src/app/components/BibPage/BibPage.jsx

```diff
@@ -40,7 +40,7 @@ const BibPage = (props) => {
   const marcRecord = isNYPLReCAP ? <MarcRecord bNumber={bNumber[0]} /> : null;
 
   return (
-    <DocumentTitle title={`${title} | Research Catalog`}>
+    <DocumentTitle title={`${title} | Shared Collection Catalog | NYPL`}>
```

Comment:
It's a "bib" for us, but for patrons it's an "item". Same idea for the breadcrumbs too.